MbDotNet/Models/Requests/SmtpRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MbDotNet.Models.Requests
+{
+    public class SmtpRequest: Request
```

Comment:
What were you referencing for the details of these request models? It mostly matches what I see in the [docs](http://www.mbtest.org/docs/protocols/smtp), but the information there is pretty vague with just "array" for the types of all the collections.

Reply:
Yes, that version of the docs is a bit outdated. What I did to generate those DTOs was creating a SMTP imposter from Postman then retrieve it. For example, this is what I just got from a MB instance after running a test:

```
{
    "protocol": "smtp",
    "port": 587,
    "name": "SMTP",
    "numberOfRequests": 1,
    "recordRequests": true,
    "requests": [
        {
            "requestFrom": "xxxxx",
            "ip": "xxxxxx",
            "envelopeFrom": "xxxxx",
            "envelopeTo": [
                "xxxxx",
                "xxxxxx"
            ],
            "from": {
                "address": "xxxxxx",
                "name": ""
            },
            "to": [
                {
                    "address": "xxxxx",
                    "name": ""
                }
            ],
            "cc": [],
            "bcc": [],
            "subject": "xxxxxx",
            "priority": "normal",
            "references": [],
            "inReplyTo": [],
            "text": "xxxxxx",
            "html": "xxxxxx",
            "attachments": [],
            "timestamp": "2021-04-15T13:42:23.461Z"
        }
    ],
    "stubs": [],
    "_links": {
        "self": {
            "href": "http://xxxxx:2525/imposters/587"
        },
        "stubs": {
            "href": "http://xxxxxx:2525/imposters/587/stubs"
        }
    }
}
```

There is also a sample of the payload for an SMTP imposter on Mountebank site [here](http://www.mbtest.org/docs/api/mocks).

Reply:
Perfect!